pages/index.js

```diff
@@ -12,7 +13,11 @@ export default function HomePage() {
     const selectedOption = formElements.option.value;
     const selectedDate = formElements.date.value;
     if (selectedOption && selectedDate) {
-      const newData = { option: selectedOption, date: selectedDate };
+      const newData = {
+        id: uuidv4(), // Eindeutige ID für jeden generierten Task als Lösung
```

Comment:
You do not need to write this as a comment. But if you need this explanation it is ok. :)

Reply:
yes, the comment was just a notice between the PR´s :) is deleted :)